registries/adapters/adapter.go

```diff
@@ -20,14 +20,15 @@ import (
 	"bytes"
 	b64 "encoding/base64"
 	"encoding/json"
-	"io/ioutil"
-	"net/http"
 	"net/url"
 	"strconv"
 
+	"io/ioutil"
+	"net/http"
+
```

Comment:
Not sure what caused these to be moved. `goimports` keeps these stdlibs in alphabetical order with the others. I'd move these 2 back to where they were.

Reply:
Yeah changed my editor so might be some settings I need to tweak